tests/rpc_server.rs

```diff
@@ -156,12 +156,13 @@ async fn test_eth_get_block_by_hash() {
 }
 
 fn get_header_with_proof() -> HeaderWithProof {
-    let file =
-        fs::read_to_string("trin-validation/src/assets/fluffy/header_with_proofs.json").unwrap();
-    let json: Value = serde_json::from_str(&file).unwrap();
-    let json = json.as_object().unwrap();
-    let raw_header = json.get("1000001").unwrap().as_object().unwrap();
-    let raw_header = raw_header.get("value").unwrap().as_str().unwrap();
-    let hwp_ssz = hex_decode(raw_header).unwrap();
-    HeaderWithProof::from_ssz_bytes(&hwp_ssz).unwrap()
+    let file = fs::read_to_string("trin-validation/src/assets/hive/blocks.yaml").unwrap();
+    let value: Value = serde_yaml::from_str(&file).unwrap();
+    let all_blocks = value.as_sequence().unwrap();
+    let post_shanghai = all_blocks.last().unwrap();
+    assert_eq!(post_shanghai["number"], 1751000);
```

Comment:
The point of this assertion right now is that if someone appends a new block into the yaml file, then this function would start testing a different block. So this assertion is a way of adding some confidence that we're testing the block that we think we are.

It probably makes sense for me to add that comment inline into the code above.

Reply:
Now that we're using yaml and comments are supported, seems like it might also be worth a trailing comment at the bottom of the yaml file